frontend/pages/index.tsx

```diff
@@ -15,20 +30,101 @@ const Home: NextPage = () => {
     first_name: firstName,
     last_name: lastName,
   } = (response?.data as unknown as UserAPIResponse) || DEFAULT_USER_API_RESPONSE;
-  return (
+
+  const [role, setRole] = React.useState('');
```

Comment:
I think we were changing this thing because the old UI for it we may not need to change page depending on role. Can you come with my chat with Peter tomorrow 9pm?

Reply:
yes I join you guys tomorrow